_schemaV2.graphql

```diff
@@ -3323,6 +3323,31 @@ type CitySponsoredContent {
   ): ShowConnection
 }
 
+type Collection implements Node {
```

Comment:
what's this from? feels like a pretty generically-named type to exist in the schema

Reply:
https://www.artsy.net/collections

It already existed I guess this union is the first time it's getting exposed?

Reply:
Actually; that doesn't seem possible, since we obviously have the UI on artsy.net